src/mm-dialog/mm-dialog.html

```diff
@@ -0,0 +1,36 @@
+<!--
+ * @license
+ * Copyright (c) 2015 MediaMath Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at http://mediamath.github.io/strand/LICENSE.txt
+
+-->
+<link rel="import" href="../../bower_components/polymer/polymer.html"/>
+<link rel="import" href="../shared/js/colors.html"/>
+<link rel="import" href="../mm-action/mm-action.html"/>
+<link rel="import" href="../mm-button/mm-button.html"/>
+<link rel="import" href="../mm-header/mm-header.html"/>
+<link rel="import" href="../mm-footer/mm-footer.html"/>
+<link rel="import" href="../mm-modal/mm-modal.html"/>
+<link rel="import" href="../shared/behaviors/stackable.html"/>
+
+<dom-module id="mm-dialog">
+	<link rel="import" type="css" href="mm-dialog.css"/>
+
+	<template>
+		<mm-modal id="dialog-inner-modal" hidden>
+			<div class="content-top">
```

Comment:
I'd give .content-top a padding of about 30px, so it feels less claustrophobic. This also follows the current T1 implementation.